src/concrete/ml/sklearn/rf.py

```diff
@@ -124,7 +124,7 @@ def load_dict(cls, metadata: Dict):
             obj.sklearn_model,
             numpy.zeros((len(obj.input_quantizers),))[None, ...],
             framework=obj.framework,
-            output_n_bits=obj.n_bits,
+            output_n_bits=obj.n_bits["op_leaves"] if isinstance(obj.n_bits, Dict) else obj.n_bits,
```

Comment:
`obj.n_bits` should only be a dict here, since you call `get_n_bits_dict_trees` in the init, so no need for the isinstance normally

same goes for all below changes

Reply:
`get_n_bits_dict_trees` is not in the init but in the fit. 
I had "test_pipline" and "grid_search" tests failing because of that.

Reply:
ah yes I see, pretty annoying then but oh well